### PR TITLE
Code lines comments

### PR DESCRIPTION
**Pull request by andreisamoila74**

### CODE REVIEW

File: codewell-fe/src/Helpers/Comment.js [hunk 1]
```diff
@@ -25,3 +25,36 @@ export function getPostCommentButtonId(lineNumber, parentCommentId) {
...
+ * the comments on the line with the index equals to the key.
+ * A code line comment is a comment that has a defined line_number.
+ */
+export function getCodeLinesComments(comments) {
+  let commentsByLine = {};
+  for (let comment of comments.filter(isCodeLineComment)) {
+    let lineComments = initProperty(commentsByLine, comment.line_number, []);
+    lineComments.push(comment);
+  }
+  return commentsByLine;
```
Comment (andreisamoila74) [inline codewell-fe/src/Helpers/Comment.js:59]: comm

Comment (andreisamoila74) [reply codewell-fe/src/Helpers/Comment.js:59]: f

Comment (andreisamoila74) [inline codewell-fe/src/Helpers/Comment.js:59]: m

File: codewell-fe/src/Helpers/Comment.js [hunk 2]
```diff
@@ -25,3 +25,36 @@ export function getPostCommentButtonId(lineNumber, parentCommentId) {
     ? `post-comment-button-codeline-${lineNumber}` : parentCommentId
     ? `post-comment-button-reply-${parentCommentId}` : "post-comment-button";
 }
+
+/**
+ * Receives a comment object as parameter and returns 1 if the comment is
+ * a code line comment, 0 otherwise.
+ * A code line comment is a comment that has a defined line_number.
+ */
+export function isCodeLineComment(comment) {
```
Comment (S7012MY) [inline codewell-fe/src/Helpers/Comment.js:34]: why do we need this function?

File: codewell-fe/src/Components/Comment/CommentCard.js [hunk 3]
```diff
@@ -38,11 +39,16 @@ function CommentCard(props) {
   }
 
   const actions = [
-    <span
-      id={"reply-button-" + props.commentIndex}
-      onClick={() => toggleReplyBoxVisibility(props.commentIndex)}>
-      {t('Reply to')}
-    </span>
+    <>
+    {props.isCodeLineComment
```
Comment (S7012MY) [inline codewell-fe/src/Components/Comment/CommentCard.js:43]: code line comments should also have replies

**Review by S7012MY — CHANGES_REQUESTED** [review]

File: codewell-fe/src/Helpers/Comment.js (hunk 2 above)
Comment (andreisamoila74) [reply codewell-fe/src/Helpers/Comment.js:34]: take a look at the last function from this helper, I used this function as a filter

**Review by S7012MY — APPROVED** [review]

**Pull request merged**